src/App.tsx

```diff
@@ -26,15 +27,17 @@ function App() {
   }
 
   return (
-    <ThemeProvider theme={theme}>
-      <MuiThemeProvider theme={theme}>
-          <UserCtx.Provider value={{user, setUser, removeUser}}>
-          <CssBaseline />
-          <Bar />
-          {user ? <Steps /> : null}
-        </UserCtx.Provider>
-      </MuiThemeProvider>
-    </ThemeProvider>
+    <SnackbarProvider>
+      <ThemeProvider theme={theme}>
+        <MuiThemeProvider theme={theme}>
+          <UserCtx.Provider value={{ user, setUser, removeUser }}>
+            <CssBaseline />
+            <Bar />
+            {user ? <Steps /> : null}
```

Comment:
🤷‍♂️ By the way you can simplify this

```suggestion
            {user && <Steps />}
```